addons/loyalty/models/loyalty_reward.py

```diff
@@ -102,6 +102,10 @@ def _compute_display_name(self):
             'The discount must be strictly positive.'),
     ]
 
+    def action_archive(self):
+        res = super().action_archive()
+        return res
+
```

Comment:
? :eyes:

Reply:
oh sorry i thought that since i am using the action_archive in a function i had to override
but then i realized it's python inheritance